other/field-types/checkbox.md

```diff
@@ -61,5 +15,7 @@ You may also want to set `contextual: true` so the field does not *also* appear
 | name | `string` | | Sets the name of the field in the database |
 | label | `string` | | Sets the label of the field that the user sees |
 | choices | `array` |  | An array of choices that the user can select from. Each must be an object with value and label properties. |  
-| showFields | `boolean` | false | A sub-property of choices, which allows additional fields to be displayed on selection |
+| *showFields* | `boolean` | false | A sub-property of choices, which allows additional fields to be displayed on selection |
```

Comment:
`showFields` is italicized but not linked.

Reply:
I see this is done on boolean as well. I don't get the meaning.

Reply:
So I italicized and called out the more important sub-properties, even though they are documented in more detail in the choices doc, so that the table could provide a stronger "at a glance" idea of what fields are needed or commonly used. Are there any other ways that we can denote important subfields?